README.md

```diff
@@ -1,8 +1,48 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, Maven, AWS, C++, C, JavaScript, HTML/CSS, Android, Docker, JSON
+
+**Familiar:** Flask, Google Cloud Platform, Blockchain, Heroku, UML, PHP
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/linux_sql)]: Developed an application with Bash, Docker, Git, and PostgreSQL to record the hardware specifications and monitor resources usages (CPU/memory, etc.) of nodes within a Linux cluster. The data collected will be stored in a PostgreSQL database and used to generate reports for resource planning with SQL queries. Crontab was effectively used to collect continuous resource usage data from a cluster, which was later used to generate the report.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/core_java)]: Grep App. Designed an application that mimics the Linux grep command with Core Java. The purpose of this application is to search through a given directory for any matches to a user-given string and write the found matching strings to an output file. The technologies used include Java 8, IntelliJ IDE, Maven, Docker, and Git.
```

Comment:
**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/core_java)]: Grep App. Designed an application that mimics the Linux grep command with Core Java. This application aims to search through a given directory for any matches to a user-given string and write the found matching strings to an output file. The technologies used include Java 8, IntelliJ IDE, Maven, Docker, and Git.